src/SFA.DAS.Forecasting.Application/Infrastructure/Registries/ConfigurationRegistry.cs

```diff
@@ -90,21 +99,20 @@ public string GetConnectionString(string name)
                 : GetSecret(connectionString).Result;
         }
 
-        public TConfig SetUpConfiguration<TConfigInterface, TConfig>(string serviceName) where TConfig : class, TConfigInterface
+        public TConfig SetUpConfiguration<TConfig>(string serviceName) where TConfig : class
         {
             var environment = Environment.GetEnvironmentVariable("DASENV");
             if (string.IsNullOrEmpty(environment))
             {
                 environment = GetAppSetting("EnvironmentName", false);
             }
 
-            var storageConnectionString = GetAppSetting("ConfigurationStorageConnectionString", true);
```

Comment:
@abroodie 
What key for CS is correct?
Had StorageConnectionString in my local and ConfigurationStorageConnectionString here.